src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -236,6 +257,22 @@ def create_privacy_request(
     )
 
 
+def _send_verification_code_to_user(
+    db: Session, privacy_request: PrivacyRequest, email: Optional[str]
+) -> None:
+    """Generate and cache a verification code, and then email to the user"""
+    verification_code: str = generate_id_verification_code()
+    privacy_request.cache_identity_verification_code(verification_code)
+    dispatch_email(
+        db,
+        action_type=EmailActionType.SUBJECT_IDENTITY_VERIFICATION,
+        to_email=email,
+        email_body_params=SubjectIdentityVerificationBodyParams(
+            access_code=verification_code
+        ),
+    )
+
```

Comment:
Verified in local integration testing, after setting identity_verification_required to True and setting up a separate mailgun `EmailConfig`, submitting a privacy request for an email address I own, sent the following to my email:

```
Your one-time code is XXXXXX. Hurry! It expires in 10 minutes.
```